images/pidfd.proto

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+
+syntax = "proto2";
+
+import "fown.proto";
+
+message pidfd_entry {
+	required uint32		id		= 1;
+	required uint32		flags		= 2;
+	required sint32		mnt_id		= 3;
+	required uint32		ino		= 4;
+	required uint32		pid		= 5;
```

Comment:
why do we need mnt_id, ino, pid? Do we use them on restore?

Reply:
We don't need them during restore.
But, we do need the contents of the Pid field during dump so I ended up creating a new struct pidfd_dump_info in order to extract the pid as well. Is this better?

Reply:
We also use the inode numbers while restoring pidfds that refer to dead processes.